controller/user_controller.py

```diff
@@ -0,0 +1,75 @@
+import os
+import psycopg2
+
+from model.user_model import User
+
+
+class UserController:
+    session = None
+    def __init__(self, session):
+        self.session = session
+
+
+    def add_user(self, user_data):
+        self.session.add(user_data)
+        try:
+            self.session.commit()
```

Comment:
i need to look at how this session works, if it works in the way i imagine, we will have race issues

Reply:
or, i think the question is why are you commiting in a non-transactional request